README.md

```diff
@@ -156,6 +156,7 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
 * [Gradle Docker plugin](https://github.com/gesellix/gradle-docker-plugin) - A Docker remote api plugin for Gradle by [@gesellix][gesellix]
 * [powerstrip](https://github.com/clusterhq/powerstrip) - A tool for prototyping Docker extensions by [@clusterhq](https://github.com/clusterhq)
 * [sbt-docker-compose](https://github.com/Tapad/sbt-docker-compose) - Integrates Docker Compose functionality into sbt by [@kurtkopchik](https://github.com/kurtkopchik/)
+* [docker-maven-plugin](https://github.com/spotify/docker-maven-plugin) - A Maven plugin for building and pushing Docker images by [@spotify](https://github.com/spotify/)
```

Comment:
yes please. docker-maven-plugin should be in line 151: https://github.com/veggiemonk/awesome-docker/pull/302/files#diff-04c6e90faac2675aa89e2176d2eec7d8R151